web/services/getHolidaySummary.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/*
+ * Copyright (C) 2021 Igalia, S.L. <info@igalia.com>
+ *
+ * This file is part of PhpReport.
+ *
+ * PhpReport is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PhpReport is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PhpReport.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+use Phpreport\Web\services\HolidayService;
+
+define('PHPREPORT_ROOT', __DIR__ . '/../../');
+
+require_once(PHPREPORT_ROOT . "/vendor/autoload.php");
+require_once(PHPREPORT_ROOT . '/util/LoginManager.php');
+
+$csvExport = array_key_exists("format", $_GET) && $_GET["format"] == "csv";
+
+$loginManager = new \LoginManager();
+
+$holidayService = new HolidayService($loginManager);
+
+$usersAndWeeks = $holidayService->retrieveHolidaysSummary();
```

Comment:
I passed "2024" as a parameter here, to test the results of a different year, and I'm still getting 2022 results. Can you check why?

Reply:
The year wasn't being passed to the method that generates the weeks per year. Can you try now, please?

I also added support to the service endpoint: `/web/services/getHolidaySummary.php?year=2024`

Reply:
Awesome, thanks!